sys/net/application_layer/dhcpv6/client.c

```diff
@@ -841,6 +841,8 @@ static void _request_renew_rebind(uint8_t type)
     if (type != DHCPV6_REBIND) {
         msg_len += _compose_sid_opt((dhcpv6_opt_duid_t *)&send_buf[msg_len]);
     }
+    msg_len += _compose_mud_url_opt((dhcpv6_opt_mud_url_t *)&send_buf[msg_len],
+                                     sizeof(send_buf) - msg_len);
```

Comment:
I think the indentation is off here a bit

```suggestion
    msg_len += _compose_mud_url_opt((dhcpv6_opt_mud_url_t *)&send_buf[msg_len],
                                    sizeof(send_buf) - msg_len);
```